### PR TITLE
Remove and restructure unused code

### PR DESCRIPTION
**Pull request by niekdejonge**

### CODE REVIEW

File: ms2query/results_table.py [hunk 1]
```diff
@@ -9,15 +9,12 @@
 class ResultsTable:
     default_columns = ["spectrum_ids",
                        "inchikey",
-                       "precursor_mz*0.001",
```
Comment (florian-huber) [inline ms2query/results_table.py:12]: Also good that here got rid of this string contruction with numbers and "*".

File: notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py [hunk 2]
```diff
@@ -3,27 +3,27 @@
 
 file_date = "15_12_2021"
 
-path_data = "C:\\HSD\\OneDrive - Hochschule Düsseldorf\\Data\\ms2query"
+path_data = "C:/Users/jonge094/PycharmProjects/PhD_MS2Query/ms2query/data/libraries_and_models/gnps_15_12_2021/"
 
 assert os.path.exists(os.path.join(path_data, "library_GNPS_15_12", f"library_GNPS_{file_date}.sqlite"))
```
Comment (florian-huber) [inline notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py:8]:
```suggestion
path_library = os.path.join(path_data, "library_GNPS_15_12")

assert os.path.exists(os.path.join(path_library, f"library_GNPS_{file_date}.sqlite"))
```

File: notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py [hunk 3]
```diff
@@ -3,27 +3,27 @@
...
-assert os.path.exists(os.path.join(path_data, "trained_models", f"spec2vec_model_GNPS_{file_date}.model"))
-assert os.path.exists(os.path.join(path_data, "trained_models", f"ms2ds_GNPS_{file_date}.hdf5"))
+assert os.path.exists(os.path.join(path_data, "library_GNPS_15_12", f"spec2vec_model_GNPS_{file_date}.model"))
+assert os.path.exists(os.path.join(path_data, "library_GNPS_15_12", f"ms2ds_model_GNPS_{file_date}.hdf5"))
 assert os.path.exists(os.path.join(path_data, "library_GNPS_15_12", f"library_GNPS_{file_date}_s2v_embeddings.pickle"))
 assert os.path.exists(os.path.join(path_data, "library_GNPS_15_12", f"library_GNPS_{file_date}_ms2ds_embeddings.pickle"))
```
Comment (florian-huber) [inline notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py:12]:
```suggestion
assert os.path.exists(os.path.join(path_library, f"spec2vec_model_GNPS_{file_date}.model"))
assert os.path.exists(os.path.join(path_library, f"ms2ds_model_GNPS_{file_date}.hdf5"))
assert os.path.exists(os.path.join(path_library, f"library_GNPS_{file_date}_s2v_embeddings.pickle"))
assert os.path.exists(os.path.join(path_library, f"library_GNPS_{file_date}_ms2ds_embeddings.pickle"))
```

File: notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py [hunk 4]
```diff
@@ -3,27 +3,27 @@
...
+assert os.path.exists(os.path.join(path_data, "in_between_files", f"GNPS_{file_date}_pos_tanimoto_scores.pickle"))
 
 collector = DataCollectorForTraining(
     os.path.join(path_data, "library_GNPS_15_12", f"library_GNPS_{file_date}.sqlite"),
-    os.path.join(path_data, "trained_models", f"spec2vec_model_GNPS_{file_date}.model"),
-    os.path.join(path_data, "trained_models", f"ms2ds_GNPS_{file_date}.hdf5"),
+    os.path.join(path_data, "library_GNPS_15_12", f"spec2vec_model_GNPS_{file_date}.model"),
+    os.path.join(path_data, "library_GNPS_15_12", f"ms2ds_model_GNPS_{file_date}.hdf5"),
     os.path.join(path_data, "library_GNPS_15_12", f"library_GNPS_{file_date}_s2v_embeddings.pickle"),
     os.path.join(path_data, "library_GNPS_15_12", f"library_GNPS_{file_date}_ms2ds_embeddings.pickle"),
```
Comment (florian-huber) [inline notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py:22]:
```suggestion
    os.path.join(path_library, f"spec2vec_model_GNPS_{file_date}.model"),
    os.path.join(path_library, f"ms2ds_model_GNPS_{file_date}.hdf5"),
    os.path.join(path_library, f"library_GNPS_{file_date}_s2v_embeddings.pickle"),
    os.path.join(path_library, f"library_GNPS_{file_date}_ms2ds_embeddings.pickle"),
```

File: notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py (hunk 2 above)
Comment (florian-huber) [reply notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py:8]: If you agree with this change, you can also click on "Add suggestion to batch" or "commit suggestion".

File: notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py [hunk 5]
```diff
@@ -3,27 +3,27 @@
...
-    os.path.join(path_data, "ms2q_train_spectra.pickle"),
-    os.path.join(path_data, "ms2q_val_spectra.pickle"),
-    os.path.join(path_data, f"GNPS_{file_date}_pos_tanimoto_scores.pickle"),
+    os.path.join(path_data, "in_between_files", "ms2q_train_spectra.pickle"),
+    os.path.join(path_data, "in_between_files", "ms2q_val_spectra.pickle"),
+    os.path.join(path_data, "in_between_files", f"GNPS_{file_date}_pos_tanimoto_scores.pickle"),
```
Comment (florian-huber) [inline notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py:25]:
```suggestion
    os.path.join(path_in_between, "ms2q_train_spectra.pickle"),
    os.path.join(path_in_between, "ms2q_val_spectra.pickle"),
    os.path.join(path_in_between, f"GNPS_{file_date}_pos_tanimoto_scores.pickle"),
```

File: tests/test_ms2library.py [hunk 6]
```diff
@@ -36,9 +36,8 @@ def file_names():
         path_to_tests_dir,
         "general_test_files/100_test_spectra_ms2ds_embeddings.pickle")
     spectrum_id_column_name = "spectrumid"
-    ms2q_model_file_name = os.path.join(
-        os.path.split(os.path.dirname(__file__))[0],
-        'tests/test_files/test_files_ms2library/ms2query_model_all_scores_dropout_regularization.hdf5')
+    ms2q_model_file_name = os.path.join(path_to_tests_dir,
+        "general_test_files/test_ms2q_rf_model.pickle")
```
Comment (florian-huber) [inline tests/test_ms2library.py:40]:
```suggestion
    ms2q_model_file_name = os.path.join(path_to_tests_dir,
        "general_test_files", "test_ms2q_rf_model.pickle")
```

**Review by florian-huber — APPROVED** [review]
Looks good. I only added a few changes to improve the readility of some parts.

**Comment by niekdejonge** [comment]
#121 is partly solved with this pull request and #125 is closed with this pull request

**Pull request merged**